src/components/App/App.js

```diff
@@ -36,10 +36,14 @@ class App extends Component {
     window.addEventListener('online', this.handleOnline);
     window.addEventListener('offline', this.handleOffline);
 
+    const isHiddenAdminPage =
+      window.location.pathname === '/admin/training' ||
+      window.location.pathname === '/admin/preview';
```

Comment:
Think this would work better

```
const isHiddenAdminPage = window.location.pathname.startsWith('/admin/');

Reply:
That's fair, I wasnt sure where we want to send things like /admin/foobar